components/external_server/src/healthcheck.py

```diff
@@ -0,0 +1,15 @@
+"""Health check."""
+
+from http import HTTPStatus
+from os import getenv
+from urllib.request import urlopen
+import sys
+
+external_server_port = getenv("EXTERNAL_SERVER_PORT", "5001")
+# pylint: disable=invalid-name
+exit_code = 1
+try:
+    with urlopen(f"http://localhost:{external_server_port}/api/health") as response:  # nosec B310
+        exit_code = 0 if response.status == HTTPStatus.OK else 1
+finally:
+    sys.exit(exit_code)
```

Comment:
I was thinking of a nice way to use `raise SystemExit` instead, but it's a bit tedious with the status NOK check

Reply:
I like this solution because it prevents a bare exception check that linters would complain about.